packages/thirdweb/src/cli/bin.ts

```diff
@@ -29,7 +29,7 @@ async function main() {
       const isBunAvailable = (() => {
         try {
           const res = execSync("bun --version", {
-            stdio: "ignore",
+            // stdio: "ignore",
```

Comment:
is this part of the fix or just left over?

Reply:
This is part of the fix, when `stdio` is set to `ignore`, `res` is `null`.